### DISCUSSION TITLE
CMS File Feed

### DISCUSSION THREAD

**Discussion by djohnson071815** [CMSgov/price-transparency-guide#472] [Q&A] [answered]
Are both the allowed amount file and in-network files required to be sent to CMS or just populated for public review on respective websites?

**Comment by olderthandirt5** [comment]
AFAIK there is no requirement to send any files to CMS and that would be a nightmare scenario for them. :-)

**Comment by shaselton-usds** [comment, marked as answer]
@djohnson071815 The files are not to be sent to CMS. You can read more about disclosure requirements from the rule [here](https://www.federalregister.gov/d/2020-24591/p-781).
